packages/login/src/components/passwordUpdate/PasswordUpdateDialog/PasswordInput.js

```diff
@@ -23,7 +25,8 @@ PasswordInput.propTypes = {
   onChange: React.PropTypes.func.isRequired,
   readOnly: React.PropTypes.bool,
   onKeyDown: React.PropTypes.func,
-  autoFocus: React.PropTypes.bool
+  autoFocus: React.PropTypes.bool,
+  formGroupStateClass: React.PropTypes.string
```

Comment:
"className" could be confusing, because of "classNames" package and JSX "className" substitute for HTML attribute "class". "formGroupStateClass" is verbose but semantic. The word "State" shall implicate changeability. The word "formGroup" could be replaced by "field", but this isn't as precise. I would prefer to leave it how it is.